test/functional/p2p-compactblocks.py

```diff
@@ -293,7 +293,7 @@ def test_compactblock_construction(self, node, test_node, version, use_witness_a
 
         # Store the raw block in our internal format.
         block = FromHex(CBlock(), node.getblock("%02x" % block_hash, False))
-        [tx.calc_sha256() for tx in block.vtx]
```

Comment:
Yes it does :-)

```shell
$ flake8 test/functional/p2p-compactblocks.py | grep redefine
test/functional/p2p-compactblocks.py:296:31: F812 list comprehension redefines 'tx' from line 276
```

Should I remove the commit 263686d03fbfa2191c160d4c50951852fa1a4900 from this PR?